lib/generator.js

```diff
@@ -137,6 +137,7 @@ function buildOperations(spec) {
       'tags',
       'consumes',
       'produces',
+      'responses',
```

Comment:
I'm concerned with adjusting the generator like this without any tests or anything.  What will this affect with other users of the generator.

Reply:
@shuowu-okta

Reply:
It's an additive change, I don't think it will break any existing generation logic. But let me figure out how to cover it with tests.